spec/lob/v1/address_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe Lob::V1::Address do
 
-  let(:sample_params) {
-    {
+  before :each do
+    @sample_params = {
```

Comment:
@charlesbjohnson no issue, I noticed we were using `before :each` in all the other tests so I wanted to change it for consistency.
(However, I think `let` actually makes more sense in this case and in [many](https://github.com/lob/lob-ruby/blob/master/spec/lob/v1/job_spec.rb) [other](https://github.com/lob/lob-ruby/blob/master/spec/lob/v1/check_spec.rb) tests. because we never change `@sample_params`, there's no need to set this variable before each test - this code just needs to run once. Doesn't make a huge difference in terms of overhead, just my thoughts.)